src/grisp_onewire.erl

```diff
@@ -115,6 +195,22 @@ write_triplet(Dir) ->
                                              {read, 1, ?I2C_M_NO_RD_ACK}]),
     {D, T, S}.
 
+%% @doc Search the bus for devices.
+%%
```

Comment:
A bit more text explaining what it returns:

List of 64bit addresses of devices on the bus (as list of 8 byte values)
The other return values respectively